fieldsolver/derivatives.cpp

```diff
@@ -825,5 +838,7 @@ void calculateScaledDeltasSimple(dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geome
       computeTimer.stop(N_cells,"Spatial Cells");
    }
 
+   
```

Comment:
Meh, stoopid githoob. I want to comment on 20 lines above, we can pare down from ALL_SPATIAL_DATA to less stuff being communicated.

Reply:
This is pretty old. Are we good with `CELL_PARAMETERS` or?

Reply:
No, you require derivatives for jperb so those are needed too. And in CELL_PARAMETERS you don't need a whole lot but I don't know whether there is flags for just the things you need, rho, v and bvol probably, but pressure I'm not sure.

Reply:
ALL_SPATIAL_DATA is:
```
  const uint64_t ALL_SPATIAL_DATA =
      CELL_PARAMETERS
      | CELL_DERIVATIVES | CELL_BVOL_DERIVATIVES
      | CELL_SYSBOUNDARYFLAG
      | POP_METADATA | RANDOMGEN;
      ```
      So I'd stick with this. (It's a bit confusingly named, but it indeed doesn't include distribution data.